### PR TITLE
Update kor - korea.txt

### PR DESCRIPTION
**Pull request by Danarca**
Includes everything from vanilla - sorry!

### CODE REVIEW

File: common/history/countries/kor - korea.txt [hunk 1]
```diff
@@ -1,7 +1,10 @@
 ﻿COUNTRIES = {
 	c:KOR = {
 		effect_starting_technology_tier_4_tech = yes
-	
+		
+		add_technology_researched = urban_planning
+		add_technology_researched = law_enforcement
+
```
Comment (HiddeLekanne) [inline common/history/countries/kor - korea.txt:8]: still needs starting laws, the list we have isn't all laws.

Comment (Danarca) [reply common/history/countries/kor - korea.txt:8]: Would a `effect_starting_politics_traditional = yes` statement work?

Comment (HiddeLekanne) [reply common/history/countries/kor - korea.txt:8]: yes that one is needed

Comment (HiddeLekanne) [reply common/history/countries/kor - korea.txt:8]: above the adjustments we make

**Review by HiddeLekanne — APPROVED** [review]

**Pull request merged**